src/GravityFormsExtensions.php

```diff
@@ -887,7 +893,7 @@ public function p4_gf_custom_confirmation_redirect($confirmation, $form, $entry)
                 }
                 </script>';
 
-            $confirmation = '<p>' . $html . '</p>' . $script;
+            $confirmation = '<p class="confirmation-redirect">' . $html . '</p>' . $script;
```

Comment:
Why is this new class needed?

Reply:
This was before I found out why the page was not redirecting automatically, I used that to run the test and see why, removing it now.